### DISCUSSION TITLE
Firebase Token

### DISCUSSION THREAD

**Discussion by fatemafreezewala** [invertase/react-native-firebase#4972] [Q&A] [answered]
Please Solve This 
 Possible Unhandled Promise Rejection (id: 1):
TypeError: _messaging.default.getToken is not a function. (In '_messaging.default.getToken()', '_messaging.default.getToken' is undefined)

**Comment by mikehardy** [comment, marked as answer]
This does not contain nearly enough information to help, sorry
If you have an issue, please open an issue (not a discussion) and provide the details requested in the template
If you would like to see a correctly created and configured example project, run https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh and examine the output.
